server/models/User.py

```diff
@@ -10,31 +11,23 @@ class User(db.Model):
     email = db.Column(db.String(128), nullable=False, unique=True)
     password = db.Column(db.String(128), nullable=False)
     isChef = db.Column(db.Boolean, nullable=False)
-    aboutMe = db.Column(db.String(600))
-    streetAddress = db.Column(db.String(200))
-    city = db.Column(db.String(50))
-    state = db.Column(db.String(50))
-    zipcode = db.Column(db.String(25))
-    country = db.Column(db.String(50))
```

Comment:
Now that we don't save all these details... we need to change the front-end to just show one address line. This is because when you open the dialog and the user already entered information, we can't reshow that exact information, because we don't store it in the database in that format

I'd do that later, but it's something to note.

Reply:
dunno why this is showing up here. this has been deleted